Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gradle:jdk11
```

Comment:
You can not it as a future change, this image will only be able to work on specific version we can make version as argument